transform/transformers/CSTransformer.py

```diff
@@ -1,123 +1,87 @@
 import dateutil.parser
-from io import BytesIO
-import os
-import shutil
-import zipfile
-
+from io import StringIO
+import os.path
 from jinja2 import Environment, PackageLoader
-
 from .ImageTransformer import ImageTransformer
 from .PCKTransformer import PCKTransformer
-from transform import settings
 
 env = Environment(loader=PackageLoader('transform', 'templates'))
 
 
 class CSTransformer(object):
+
+    receipt_path = os.getenv("SDX_FTP_RECEIPT_PATH", "EDC_QReceipts")
```

Comment:
These are in settings.py, do they need to be done here?

Reply:
Good catch , thanks , changing it